### ISSUE TITLE
Write up proposal for MDN RFCs

### ISSUE THREAD

**Issue by chrisdavidmills** [mdn/pab#82]
Including template, guidelines, a process, and a location to put it all.

**Comment by chrisdavidmills** [comment]
@Elchi3 might be able to help with this too.

**Comment by Elchi3** [comment]
Yes, thanks Chris! I initially proposed this and Dom was so kind to bring it to the MDN PAB.

Referencing here my prime example of an MDN RFC from three years back:
https://discourse.mozilla.org/t/mdn-rfc-001-mdn-wiki-pages-shouldnt-be-a-distributor-of-polyfills

**Comment by dontcallmedom** [comment]
@Elchi3 & @chrisdavidmills iterated based on discussions about opportunity assessments, which resulted in a first RFC being submitted https://github.com/mdn/content/pull/3011

I think completing this would involve documenting (e.g. in https://github.com/mdn/content/tree/main/rfcs) the format for these RFCs, and some expectations about who would be asked for comments, with what kind of review periods

**Comment by chrisdavidmills** [comment]
Documentation for the RFC format, expectations, etc. Written up at https://docs.google.com/document/d/1noj3DuKlSh33B25JgS6npav4N1kPjzmH_V5SuqP2kGw/edit

Feedback please!

**Comment by Elchi3** [comment]
Looks good, make it a PR? 

I guess the question around review periods isn't answered but we can follow up on that.

**Comment by chrisdavidmills** [comment]
> Looks good, make it a PR?
> 
> I guess the question around review periods isn't answered but we can follow up on that.

Yup! Submitted at https://github.com/mdn/content/pull/3412.
